app/assets/javascripts/editor/submissions.js

```diff
@@ -198,28 +198,6 @@ CodeOceanEditorSubmissions = {
     }
   },
 
-  submitCode: function(event) {
```

Comment:
This method is still used by the community solution (`app/assets/javascripts/community_solution.js:23`), so that the removal will break this part. Rather than just keeping it, we might also consider moving this code closer to the community solution (since it is used there).